schemas/src/test/java/com/scottlogic/deg/schemas/v0_1/ProfileValidatorTest.java

```diff
@@ -0,0 +1,161 @@
+package com.scottlogic.deg.schemas.v0_1;
+
+import com.scottlogic.deg.schemas.common.ValidationResult;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+
+class ProfileValidatorTest {
```

Comment:
Minor point, class name _should_ be pluralised, e.g. `ProfileValidatorTests`

Reply:
done